NitroxServer/Communication/Packets/Processors/PDAScanProgressPacketProcessor.cs

```diff
@@ -0,0 +1,42 @@
+using NitroxModel.DataStructures;
+using NitroxModel.Packets;
+using NitroxServer.Communication.Packets.Processors.Abstract;
+using NitroxServer.GameLogic;
+using NitroxServer.GameLogic.Entities;
+using NitroxServer.GameLogic.Unlockables;
+
+namespace NitroxServer.Communication.Packets.Processors;
+
+public class PDAScanProgressPacketProcessor : AuthenticatedPacketProcessor<PDAScanProgress>
+{
+    private readonly PlayerManager playerManager;
+    private readonly PDAStateData pdaStateData;
+    private readonly EntityRegistry entityRegistry;
+    private readonly SimulationOwnershipData simulationOwnershipData;
+
+    public PDAScanProgressPacketProcessor(PlayerManager playerManager, PDAStateData pdaStateData, EntityRegistry entityRegistry, SimulationOwnershipData simulationOwnershipData)
+    {
+        this.playerManager = playerManager;
+        this.pdaStateData = pdaStateData;
+        this.entityRegistry = entityRegistry;
+        this.simulationOwnershipData = simulationOwnershipData;
+    }
+
+    public override void Process(PDAScanProgress packet, Player player)
+    {
+        if (pdaStateData.UpdateScanProgress(packet.Id, packet.TechType, packet.Progress))
+        {
+            playerManager.SendPacketToOtherPlayers(packet, player);
+        }
+
+        if (packet.Destroy)
+        {
+            pdaStateData.FinishScanProgress(packet.Id, packet.TechType, packet.Destroy, false);
+            entityRegistry.RemoveEntity(packet.Id);
+            if (simulationOwnershipData.RevokeOwnerOfId(packet.Id))
+            {
+                playerManager.SendPacketToAllPlayers(new SimulationOwnershipChange(packet.Id, ushort.MaxValue, SimulationLockType.TRANSIENT));
+            }
+        }
```

Comment:
We need to do the same steps as the `EntityDestroyedPacketProcessor`.  You may want to refactor that out into a common place.  The main thing missing is if it is a WorldEntity, we want to stop tracking it. 

(Same for the class above)